x-pack/plugins/apm/public/components/app/transaction_details/waterfall_with_summary/waterfall_container/Waterfall/waterfall_item.tsx

```diff
@@ -216,27 +211,69 @@ export function WaterfallItem({
         </SpanActionToolTip>
         <HttpStatusCode item={item} />
         <NameLabel item={item} />
-        {errorCount > 0 && item.docType === 'transaction' ? (
-          <ErrorOverviewLink
-            serviceName={item.doc.service.name}
-            query={{
-              kuery: `${TRACE_ID} : "${item.doc.trace.id}" and transaction.id : "${item.doc.transaction.id}"`,
-            }}
-            color="danger"
-            style={{ textDecoration: 'none' }}
-          >
-            <EuiToolTip content={tooltipContent}>
-              <ErrorCount count={errorCount} />
-            </EuiToolTip>
-          </ErrorOverviewLink>
-        ) : null}
+
+        <EventOutcome
+          item={item}
+          transactionErrorCount={transactionErrorCount}
+        />
+
         <Duration item={item} />
         {item.docType === 'span' && <SyncBadge sync={item.doc.span.sync} />}
```

Comment:
```suggestion
        <Duration item={item} />
        <EventOutcome
          item={item}
          transactionErrorCount={transactionErrorCount}
        />
        {item.docType === 'span' && <SyncBadge sync={item.doc.span.sync} />}
```
I prefer the badges to be displayed after the duration (like other badges we display, which will ensure they line up).

<img width="424" alt="CleanShot 2021-08-25 at 15 17 55@2x" src="https://user-images.githubusercontent.com/4104278/130797629-6ba71244-4a0d-4e02-b6b0-6c6888e41b73.png">